articles/overview/understanding-quantum-computing.md

```diff
@@ -0,0 +1,60 @@
+
+Think of flipping a coin in the air and catching it. While it’s in the air turning over and over, it’s neither heads nor tails, but can be seen as a linear combination of all of its possible states at once – heads, tails, 75% heads, 12.5% tails, and so on.
+
+A quantum particle such as an electron has its own “heads or tails” property called **spin**, which can be either up or down (or positive or negative for a photon), or to make it more relatable to classical binary computing, let’s just say 1 or 0. When a quantum particle is in a superposition state, it’s a linear combination of all of its possible states between 1 and 0, but you don’t know which one it will be until you actually look at it, which brings up our next phenomenon, **quantum measurement**.
+
+### Quantum measurement
+
+Let’s say you want to track the height of a five-year old child every year by measuring them and placing a mark on the wall. You would expect their height to increase every year, and the mark on the wall to gradually get higher.
+
+But what if you measured that five-year old and they just stopped growing? That’s sort of what happens when you measure a quantum particle. Just the act of observing or measuring a quantum particle **collapses** the superposition state (also known as **decoherence**) and the particle takes on a classical binary state of either 1 or 0.
+
+This is helpful to us, because in computing you can do lots of things with 1’s and 0’s. However, once a quantum particle has been measured and collapsed, it stays in that state forever and will always be a 1 or 0, and cannot be used for more calculations. As you’ll see later, though, in quantum computing there are operations that can “reset” a particle back to a superposition state so it can be useful again.
+
+### Entanglement
+
+Possibly the most interesting phenomenon of quantum physics is the ability of two or more quantum particles to become **entangled** with each other. When particles become entangled, they form a single system such that the quantum state of any one particle cannot be described independently of the quantum state of the other particles. This means that whatever operation or process you apply to one particle immediately takes effect on the other particles as well.
+
+In addition, particles can maintain this connection even when separated over incredibly large distances, even light-years. The effects of quantum measurement still apply to entangled particles, such that when one particle is measured and collapses, the other particle collapses as well. However, the second one always collapses in the *opposite* state as the first one – this particular property is very helpful in quantum computing. These unique properties of entangled particles can be used to transmit quantum information from one place to another by a process called **teleportation**, and may play a critical role in future cryptography applications.
```

Comment:
> The effects of quantum measurement still apply to entangled particles, such that when one particle is measured and collapses, the other particle collapses as well. However, the second one always collapses in the *opposite* state as the first one – this particular property is very helpful in quantum computing.

This is not true in general; for example, if two qubits are prepared in the (|00⟩ + |11⟩) / √2 state, then both measurements will always agree with each other. 

> These unique properties of entangled particles can be used to transmit quantum information from one place to another by a process called **teleportation**, and may play a critical role in future cryptography applications.

This seems to conflate QKD with quantum teleportation. While some QKD protocols (e.g.: Ekert94) use teleportation, not all do. The most common one, BB84, doesn't even use multiple qubits.